pubsub/google/cloud/pubsub/message.py

```diff
@@ -85,7 +86,10 @@ def from_api_repr(cls, api_repr):
         :rtype: :class:`Message`
         :returns: The message created from the response.
         """
-        data = base64.b64decode(api_repr.get('data', b''))
+        try:
+            data = base64.b64decode(api_repr.get('data', b''))
+        except (binascii.Error, TypeError):
+            data = api_repr.get('data', b'')
```

Comment:
@dhermes The [docs](https://github.com/GoogleCloudPlatform/google-cloud-python/issues/2513#issuecomment-252433831) say it's required but the console assumes that you must b64encode it yourself before pasting it into the input?